Test/Web/FunctionalTests/FunctionalTests/IisExpress/IISExpress.cs

```diff
@@ -94,7 +94,14 @@ public void Stop()
             if (false == this.hostProcess.WaitForExit(processExitWaitTimeout))
             {
                 Trace.TraceWarning("iisexpress.exe process hasn't exited during expected time, terminating!");
-                this.hostProcess.Kill();
+                try
+                {
+                    this.hostProcess.Kill();
+                }
+                catch (Win32Exception e)
```

Comment:
If the process is already in the middle is exiting, it is guaranteed to throw a Security Exception.
From MSDN:
"_If the call to the Kill method is made while the process is currently terminating, a Win32Exception is thrown for Access Denied_."

Instead of increasing our timeout further, its better to catch